### PR TITLE
Update 2FA to MFA in Cloud articles

### PR DESCRIPTION
**Pull request by eshanrnh**
2FA is now changed to MFA in Cloud portal. Updated articles to reflect the change.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -496,7 +496,7 @@
   * [Media](umbraco-cloud/set-up/media/README.md)
     * [Connect to Azure Storage Explorer to upload files manually](umbraco-cloud/set-up/media/connect-to-azure-storage-explorer.md)
   * [Users](umbraco-cloud/set-up/users-on-cloud.md)
-  * [2-factor authentication](umbraco-cloud/set-up/2-factor-authentication-on-cloud.md)
+  * [Multi-Factor Authentication](umbraco-cloud/set-up/2-factor-authentication-on-cloud.md)
```
Comment (sofietoft) [inline SUMMARY.md:499]: This file is not being used anymore, so don't worry about updating this.

File: umbraco-cloud/SUMMARY.md [hunk 2]
```diff
@@ -51,7 +51,7 @@
 * [Media](set-up/media/README.md)
   * [Connect to Azure Storage Explorer to upload files manually](set-up/media/connect-to-azure-storage-explorer.md)
 * [Users](set-up/users-on-cloud.md)
-* [2-factor authentication](set-up/2-factor-authentication-on-cloud.md)
+* [Multi-Factor Authentication](set-up/2-factor-authentication-on-cloud.md)
```
Comment (sofietoft) [inline umbraco-cloud/SUMMARY.md:54]: Should we consider updating the URL as well? 🤔

**Review by sofietoft — COMMENTED** [review]
Looks good. I've added a couple of comments 💪

**Review by jonat123 — APPROVED** [review]
@eshanrnh I don't have anything else to add, looks good to me :D

File: umbraco-cloud/SUMMARY.md (hunk 2 above)
Comment (eshanrnh) [reply umbraco-cloud/SUMMARY.md:54]: Yup, will rename the file and update the URLs

**Pull request merged**